spec/0.0.2/README.md

```diff
@@ -262,33 +262,41 @@ Field Name | Type | Description
 
 #### <a name="artifactsObject"></a>Artifacts Object
 
-The Artifacts Object describes a list of provider specific artifact items. These artifact items will be used during installation of the containerapp to deploy it to the provider. Each artifact item is a `URL`.
+The Artifacts Object describes a list of provider specific artifact items. These artifact items will be used during installation of the containerapp to deploy it to the provider. Each provider key contains a list of artifacts. Each artifact list item is either a `URL` string or a [source control repository object](#repositoryObject).
+
+* URL: must be a URL string prepended by type such as `http://`, `https://` or `file://`. URL type `file://` may be a single file or a directory path to multiple files. Directories must end with a trailing slash such as `file://relative/path/to/multiple/artifact/files/`.
```

Comment:
This is incorrect in our examples and atomicapp now - local file should have only `file:` because `file://` implies absolute path. 

Also I would rather not supported directories - I think the `URL` should be consistent across all protocols